package/MDAnalysis/lib/log.py

```diff
@@ -335,6 +361,19 @@ def update(self, step, **kwargs):
 
         self.numouts += 1
 
+        # Update time tracking and estimate time to completion
+        self._last_time = datetime.datetime.now()
```

Comment:
To test this just check that the values are set when you call update.

```python
pm.upate()
# store values
pm.update()
# check start_time wasn't overwritten.
```

That should be enough to check that it functions correct.